### DISCUSSION TITLE
[Feat] New rate unit - messages per second

### DISCUSSION THREAD

**Discussion by allamiro** [grafana/grafana#36999] [General]
Please include this information:
### What did you do? 
created a panel for kafka messages per scond throughput  ,  then go to axes tab select unit , data rate  no messages per second option available 
### Include raw network request & response: get by opening Chrome Dev Tools (F12, Ctrl+Shift+I on windows, Cmd+Opt+I on Mac), go the network tab.
 
I propose this as an enhancement

**Comment by hhromic** [comment]
This is a 3-year old request and still very relevant.
I came looking for this unit after the exact same use-case: panels showing Kafka topic input/output message throughputs.
I was amused to discover that Grafana has so many useful built-in units except this one. Not only "messages/sec" but also "events/sec" or "items/sec" is missing imho.

**Comment by hhromic** [comment]
There is no interest from the Grafana team to implement this simple request? :(

**Comment by hhromic** [comment]
I implemented this in the following PR: https://github.com/grafana/grafana/pull/70726
Which is slated to appear in Grafana 10.1.x :)

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
